hack-kni/legacy/nrtpatch.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/go-logr/logr"
+
+	topologyv1alpha2 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha2"
+	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
+	"sigs.k8s.io/scheduler-plugins/pkg/noderesourcetopology/nodeconfig"
+)
+
+func toTopologyManagerPolicy(conf nodeconfig.TopologyManager) string {
```

Comment:
we can't change the `nodeconfig` package without either sending a u/s PR (and pulling down the changes) or forking it, so I'm doing it here